config/initializers/elasticsearch.rb

```diff
@@ -1,11 +1,18 @@
 require 'typhoeus'
 require 'typhoeus/adapters/faraday'
 
-user = CheckConfig.get('elasticsearch_user').to_s
-password = CheckConfig.get('elasticsearch_password').to_s
-host = CheckConfig.get('elasticsearch_host').to_s + ':' + CheckConfig.get('elasticsearch_port').to_s
-client = Elasticsearch::Client.new(log: (CheckConfig.get('elasticsearch_log').to_i == 1), user: user, password: password, host: host) do |f|
-  f.adapter :typhoeus
+unless ENV['ELASTICSEARCH_URL'] == ""
```

Comment:
Please use a stronger check: `unless ENV['ELASTICSEARCH_URL'].blank?`.

Reply:
Ah, good call. `.empty?` would not work right, but `.blank?` should do the trick!